tests/Microsoft.DotNet.Framework.Docker.Tests/SdkOnlyImageTests.cs

```diff
@@ -22,14 +22,28 @@ public SdkOnlyImageTests(ITestOutputHelper outputHelper)
 
         protected override string ImageType => "sdk";
 
-        public static IEnumerable<object[]> GetImageData() =>
-            ImageTestHelper.ApplyImageDataFilters(TestData.GetImageData(), "sdk");
+        public static IEnumerable<object[]> GetImageData()
+        {
+            IEnumerable<object[]> imageData =
+                ImageTestHelper.ApplyImageDataFilters(TestData.GetImageData(), ImageTypes.Sdk);
+            if (!imageData.Any())
```

Comment:
Would it make sense to move this into the `ImageTestHelper.ApplyImageDataFilters` method? This would eliminate the duplication between the SDK and WCF tests.  If you think it would be harmful to have for the other test classes, you could condition the logic on an optional parameter - e.g. "allowEmptyResults"